core/mixins/_media-queries.scss

```diff
@@ -97,3 +105,16 @@
     @content;
   }
 }
+
+
+/**
+ * Retina.
+ */
+
+@mixin respond-retina($resolution: $retina-resolution) {
```

Comment:
Yeah overlooked the current name, hmmm much prefer `hidpi` or just after the media feature: `resolution`, oh well put it down for V3.